docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: docs
+title: Why TestCafe?
+permalink: /documentation/why-testcafe/
+---
+
+# Why TestCafe?
+
+TestCafe excels where other end-to-end testing solutions fall short.
+
+## Comprehensive and easy to set up
+
+TestCafe does not depend on Selenium or other testing software. TestCafe runs on the popular Node.js platform, and makes use of the browsers that you already have. For many users, the installation process consists of a single step:
+
+```js
+npm install -g testcafe
+```
+
+## Human-readable JavaScript tests
+
+TestCafe is a JavaScript framework. But you don’t need to be a JavaScript guru to write TestCafe tests. The core components of our API are intuitive and only require a basic knowledge of the language.
```

Comment:
> Sentences should rarely if ever start with conjunctions.
Except when it's necessary to evoke a specific emotional response, which, I would argue, is the case here.

Moreover, TestCafe doesn't have a simple API. *Some parts* of our API are simple to *use*.

Reply:
I'd also advise against pushing TestCafe Studio to our open-source customers (the primary audience of this article) in such an aggressive manner.

P.S.
a) TestCafe Studio is *not* an IDE. It can't be a development environment if there's no programming involved.
b) We should probably altogether refrain from using the IDE metaphor because IDEs don't have the best reputation. I'd wager most people associate IDEs with outdated, bloated applications like Eclipse.
c) One could argue that TestCafe Studio is not as complete as it could be :)

Reply:
Would this work?

```suggestion
The core components of our Node.js API are intuitive and only require a basic knowledge of JavaScript.
```

Reply:
Looks good to me. Thanks